src/raw-value.ts

```diff
@@ -239,6 +264,73 @@ export function reverseProxyFactory(env: MembraneBroker) {
             }
             return env.getRawValue(sec);
         }
+        has(shadowTarget: ReverseShadowTarget, key: PropertyKey): boolean {
+            const { target } = this;
+            // todo: protect against leaking symbols
+            return key in target;
+        }
+        ownKeys(shadowTarget: ReverseShadowTarget): PropertyKey[] {
+            const { target } = this;
+            // todo: protect against leaking symbols
+            return ownKeys(target);
+        }
+        isExtensible(shadowTarget: ReverseShadowTarget): boolean {
+            if (!ReflectIsExtensible(shadowTarget)) {
+                return false;
+            }
+            const { target } = this;
+            if (!ReflectIsExtensible(target)) {
+                lockShadowTarget(shadowTarget, target);
+                return false;
+            }
+            return true;
+        }
+        getOwnPropertyDescriptor(shadowTarget: ReverseShadowTarget, key: PropertyKey): PropertyDescriptor | undefined {
+            const { target } = this;
+            // todo: leaking key as symbol
+            let rawDesc = ReflectGetOwnPropertyDescriptor(shadowTarget, key);
+            if (!isUndefined(rawDesc)) {
+                return rawDesc;
+            }
+            const secDesc = ReflectGetOwnPropertyDescriptor(target, key);
+            if (isUndefined(secDesc)) {
+                return secDesc;
+            }
+            rawDesc = getRawDescriptor(secDesc);
+            if (secDesc.configurable == false) {
+                // updating the descriptor to non-configurable on the shadow
+                ReflectDefineProperty(shadowTarget, key, rawDesc);
+            }
+            return rawDesc;
+        }
+        getPrototypeOf(shadowTarget: ReverseShadowTarget): RawValue {
+            const { target } = this;
+            return env.getRawValue(ReflectGetPrototypeOf(target));
+        }
+        setPrototypeOf(shadowTarget: ReverseShadowTarget, prototype: RawValue): boolean {
+            const { target } = this;
+            return ReflectSetPrototypeOf(target, env.getSecureValue(prototype));
+        }
+        preventExtensions(shadowTarget: ReverseShadowTarget): boolean {
```

Comment:
Is there a case where `preventExtensions` could return `false`, as is the trap doesn't do that.

Reply:
I think you're right. If the target is a proxy from the sandbox (manually create) that returns false on that trap, then this will not reflect that. I will fix it.

Reply:
fixed!